node/src/service.rs

```diff
@@ -26,12 +26,13 @@ use cumulus_client_service::{
 	prepare_node_config, start_collator, start_full_node, StartCollatorParams, StartFullNodeParams,
 };
 use cumulus_primitives_core::{
-	relay_chain::v1::{Hash as PHash, PersistedValidationData},
+	relay_chain::v2::{Hash as PHash, PersistedValidationData},
```

Comment:
which upstream issues are associated with the changes in the client ?

Reply:
[Companion for primitives consolidation into v2](https://github.com/paritytech/cumulus/pull/1071)